water_mark/etl_water_mark.py

```diff
@@ -0,0 +1,104 @@
+###############################################################################
+#################      _____ _____ _         ##################################
+##################    | ____|_   _| |         #################################
+###################   |  _|   | | | |          ################################
+####################  | |___  | | | |___        ###############################
+##################### |_____| |_| |_____|        ##############################
+####################                            ###############################
+################### George Meier 16 July 2019  ################################
+##################  mater_mark ETL routine    #################################
+########################################################################80chars
+#
+# This program sets up the current directory for water_mark to be run by
+# creating an input directory, and filling it with VV and  VH sar tifs
+# downloaded from asf HyP3.
+#
+###############################################################################
+#
+# Python3
+# Windows admin powershell or Linux
+# - water_mark:
+#   - water_mark.py
+#   - etl_water_mark.py
+#   - download-all-<nums>.py (python script from asf HyP3)
+#   - identify_water.py
+#   - inputs: (input directory made automatically)
+#
+################################################################################
+
+import platform
+import os
+import shutil
+import subprocess
+import zipfile
+from typing import Tuple
+
+
+def determine_OS() -> Tuple[bool, bool]:
+    if platform.system() == 'Windows':
+        return (True, False)
+    else:
+        return (False, True)
+
+
+def get_SAR_from_HyP3() -> None:
+    linuxMode, windowsMode = determine_OS()
+    python = 'python'
+    if linuxMode:
+        python = 'python3'
+    scriptToRun = 'temp'
+    for fileName in os.listdir():
+        if 'download' in fileName:
+            scriptToRun = fileName
```

Comment:
You can use https://docs.python.org/3.5/library/importlib.html#importlib.import_module instead. This will make your code much more robust and remove the need for checking OS entirely